extract_unique_sensors_01.py

```diff
@@ -0,0 +1,28 @@
+from urllib.request import urlopen
+
+sensors_data = urlopen('http://monctl.devm.camlab.kat.ac.za/kat/doc/manuals/sensors/SensorList.txt')
```

Comment:
Perhaps,
- Add the URL as a variable instead of hard coding it.
- Add try/except to handle broken links or else this will not stop until time-out and breaks the code.
See: https://wiki.python.org/moin/HandlingExceptions
- This can be a function that takes in a URL and returns a list of somethings.

Reply:
See this:


![image](https://user-images.githubusercontent.com/7910856/77741163-1a8c8280-701d-11ea-997e-06cddf286e0d.png)